server/procedure_collection.cc

```diff
@@ -99,6 +100,45 @@ initRoutes(RouteManager & manager)
     ProcedureRunCollection::initRoutes(*runManager);
 
     manager.childRoutes["runs"] = runManager;
+
+  RestRequestRouter::OnProcessRequest getLatestRun =
+      [=] (RestConnection & connection, const RestRequest & req,
+           const RestRequestParsingContext & context)
+    {
+        // load current procedure
+        Utf8String redirect("/v1/procedures/");
+        auto collection = manager.getCollection(context);
+        auto key = manager.getKey(context);
+        auto procedure = static_cast<Procedure *>(
```

Comment:
@guyd static_cast is faster than dynamic_cast. 
static_cast is when you know what you are doing. If it fails, it crashes.
dynamic_cast is when you are not sure. If it fails, it retuns a nullptr.